pkg/block/index.go

```diff
@@ -308,20 +309,21 @@ func GatherIndexIssueStats(logger log.Logger, fn string, minTime int64, maxTime
 			}
 
 			c0 := chks[i-1]
+
+			// Chunk order within block.
 			if c.MinTime > c0.MaxTime {
 				continue
 			}
 
-			// Chunks overlaps or duplicates.
 			if c.MinTime == c0.MinTime && c.MaxTime == c0.MaxTime {
-				ca := crc32.Checksum(c0.Chunk.Bytes(), castagnoli)
-				cb := crc32.Checksum(c.Chunk.Bytes(), castagnoli)
```

Comment:
what is the reason for removing the checksum?

Reply:
The reason is stated in comment -> we don't have access to chunk in this method, we are just checking the index.